File: PlayerUI/Flutter/objc/PlayerFlutter/BCOVFlutterPlugin.m
//
//  BCOVFlutterPlugin.m
//  PlayerFlutter
//
//  Created by Carlos Ceja.
//

#import "BCOVFlutterPlugin.h"

#import "BCOVVideoPlayer.h"


@implementation BCOVFlutterPlugin

+ (void)registerWithRegistrar:(nonnull NSObject<FlutterPluginRegistrar> *)registrar
{
    BCOVVideoPlayerFactory *factory = [[BCOVVideoPlayerFactory alloc] initWithRegistrar:registrar];
    [registrar registerViewFactory:factory withId:@"bcov.flutter/player_view"];
}

@end

File: PlayerUI/Flutter/swift/PlayerFlutter/BCOVFlutterPlugin.swift
//
//  BCOVFlutterPlugin.swift
//  PlayerFlutter
//
//  Created by Carlos Ceja.
//

import Foundation

import Flutter


class BCOVFlutterPlugin : NSObject, FlutterPlugin {
    
    public static func register(with registrar: FlutterPluginRegistrar) {
        let factory = BCOVVideoPlayerFactory(registrar: registrar)
        registrar.register(factory, withId: "bcov.flutter/player_view")
    }

}
